### PR TITLE
GPC-NONE: Add grafana ADR

### PR DESCRIPTION
**Pull request by oskarwilliams378**

### CODE REVIEW

File: doc/architecture/decisions/0015-grafana.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+   through our own self-hosted Grafana
+
+In contrast, the self-hosted Grafana does require some initial set up and continued maintenance, however this is
+expected to be fairly minor and easy to complete. Also, the self-hosted Grafana still has all the necessary access to
+our AMP for things like alert silencing and monitoring (for alerts defined in our AMP), and allows for a much greater
+flexibility if we desire in the future to add in more complex configuration.
+
+## Future risks
+
+The major risks here are the maintenance of the self-hosted Grafana instance that may be performed by AMG automatically.
```
Comment (olevett) [inline doc/architecture/decisions/0015-grafana.md:40]: Is Grafana included in our automated dependency upgrades?

Comment (oskarwilliams378) [reply doc/architecture/decisions/0015-grafana.md:40]: YEs

File: doc/architecture/decisions/0015-grafana.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+We expect this to be low risk and cost, as we expect any major breaking updates to require our input for AMG anyway, and
+for minor updates the maintenance should be trivial.
```
Comment (ethanmills) [inline doc/architecture/decisions/0015-grafana.md:42]: Worth mentioning that AlertManager is still AWS managed so there's limited risk to alerting?

**Review by ethanmills — APPROVED** [review]

File: doc/architecture/decisions/0015-grafana.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# 15. Grafana
```
[the remark this answers is not in the document]
Comment (oskarwilliams378) [reply doc/architecture/decisions/0015-grafana.md]: Done

File: doc/architecture/decisions/0015-grafana.md (hunk 2 above)
Comment (oskarwilliams378) [reply doc/architecture/decisions/0015-grafana.md]: Done

**Pull request merged**